src/hooks/book/use-book-reviews.ts

```diff
@@ -129,6 +130,7 @@ export const useBookReviews = (isbn: string | undefined) => {
   return {
     reviews,
     ratings,
+    // Convert the raw userRating value to a Rating object when returning
     userRating: userRating > 0 ? new Rating(userRating) : new Rating(0),
```

Comment:
Then conversion isn't needed.

```suggestion
    userRating,
```